cognite/powerops/custom_modules/power_model_v1/data_models/views/function_inputs/ShopPartialBidCalculationInput.view.yaml

```diff
@@ -43,28 +43,15 @@ properties:
   shopResults:
```

Comment:
Should this here be called `shopResultPriceProd` and is this then replacing `shopResult` as a property? If yes, how do we get information alerts from the shop results? Maybe alerts are not needed inside a shop result.

Reply:
I agree in the name change. The type SHOPResultPriceProd inherits all properties from SHOPResult, which contains alerts, so the input will still have alerts with this new type 👍